scripts/create_singularities

```diff
@@ -78,7 +79,7 @@ class Builder:
 
     @staticmethod
     def get_last_docker_version_tag(dh: str, only_good_versions: bool=False) -> Optional[tuple[str, str]]:
-        r = requests.get(f"https://registry.hub.docker.com/v2/repositories/{dh}/tags")
+        r = retry_get(f"https://registry.hub.docker.com/v2/repositories/{dh}/tags")
```

Comment:
it seems that there is a "standard" way to get retries done with requests: https://requests.readthedocs.io/en/latest/user/advanced/?highlight=Retry#example-automatic-retries but it seems to be also not that straightforward... so I am ok with this "ad-hoc" solution for now. Thank you!